Formula/vowpal-wabbit.rb

```diff
@@ -16,10 +16,10 @@ class VowpalWabbit < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "flatbuffers" => :build
   depends_on "rapidjson" => :build
   depends_on "spdlog" => :build
   depends_on "boost"
+  depends_on "eigen"
```

Comment:
This can probably be a build dependency if it’s even used.

Reply:
This dependency is not used yet, but will likely be in the next release

Reply:
If it's not even being used yet it's then we shouldn't have added it.

Reply:
It is a dependency for a new feature which has been included in this release but is disabled